packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -76,6 +76,12 @@ class WalletsController {
     receivingAddressNumber: number
     changeAddressNumber: number
   }): Promise<ChannelResponse<Wallet>> => {
+    if (!WalletsController.verifyPasswordComplexity(password)) {
+      return {
+        status: ResponseCode.Fail,
+        msg: 'At least 8 characters, three types of uppercase and lowercase alphabets, numbers, and special symbols!',
```

Comment:
To make the verification and message more clear, we should separate the results. e.g., check these:

* password !== confirmPassword,
* password length < required length
* password complexity < required conditions